src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.content {
+  font-family: "Roboto" , sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 #0000000d;
+  max-width: 100%;
+  padding: 0 50px;
+}
+
+.logo {
+  padding-top: 5px;
+}
+
+.menu_list {
+  display: flex;
+}
+
+.nav_item {
+  display: flex;
+  padding-left: 20px;
+}
+
+.nav_item :first-child {
+  padding-left: 0;
+}
+
+.nav_link {
+  padding: 23px 0;
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  color: #000;
+  transition: all 0.5s;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+}
+
+.nav_link.is_active {
+  position: relative;
+}
+
+.is_active.nav_link {
+  color: #00acdc;
+}
+
+.is_active ::after {
+  content: "";
+  position: absolute;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  bottom: 0;
+  border-radius: 8px;
+}
+
+.nav_link:hover {
+  color: #00acdc;
+  text-decoration: underline rgb(55, 103, 167) 2px;
+  transition: all 0.8s;
```

Comment:
No need to set it here. You've already set it for the default state of nav_link. Also, why the transition duration is different here?